buildkite/benchmark/run.py

```diff
@@ -50,6 +53,53 @@
             "ARROW_SRC": f"{build_dir}/arrow",  # required by Java Script benchmarks
         },
     },
+    {
+        "benchmarkable_type": "arrow-commit",
+        "repo": "https://github.com/voltrondata-labs/arrowbench.git",
+        "root": "arrowbench",
+        "branch": "main",
+        "setup_commands": [],
+        "path_to_benchmark_groups_list_json": "arrowbench/inst/benchmarks.json",
+        "url_for_benchmark_groups_list_json": "https://raw.githubusercontent.com/voltrondata-labs/arrowbench/main/inst/benchmarks.json",
+        "setup_commands_for_lang_benchmarks": {  # These commands need to be defined as functions in buildkite/benchmark/utils.sh
+            "R": [
+                "build_arrow_r",
+                "install_arrowbench",
+                "create_data_dir",
+            ],
+        },
+        "env_vars": {
+            "PYTHONFAULTHANDLER": "1",  # makes it easy to debug segmentation faults
+            "BENCHMARKS_DATA_DIR": f"{os.getenv('HOME')}/data",  # allows to avoid loading Python and R benchmarks input data from s3 for every build
+            "ARROWBENCH_DATA_DIR": f"{os.getenv('HOME')}/data",  # allows to avoid loading R benchmarks input data from s3 for every build
+        },
+    },
+    {
+        "benchmarkable_type": "arrow-commit",
+        "repo": "https://github.com/voltrondata-labs/arrow-benchmarks-ci.git",
+        "root": "arrow-benchmarks-ci/adapters",
+        "branch": "edward/direct-running",  # "main", # TODO: switch back to main
+        "setup_commands": ["pip install -r requirements.txt"],
+        "path_to_benchmark_groups_list_json": "arrowbench/inst/benchmarks.json",
+        "url_for_benchmark_groups_list_json": "https://raw.githubusercontent.com/voltrondata-labs/arrowbench/main/inst/benchmarks.json",
+        "setup_commands_for_lang_benchmarks": {  # These commands need to be defined as functions in buildkite/benchmark/utils.sh
+            "C++": ["install_archery"],
+            "Python": ["create_data_dir"],
+            "R": [
+                "build_arrow_r",
+                "install_arrowbench",
+                "create_data_dir",
+            ],
+            "Java": ["build_arrow_java", "install_archery"],
+            "JavaScript": ["install_java_script_project_dependencies"],
+        },
+        "env_vars": {
+            "PYTHONFAULTHANDLER": "1",  # makes it easy to debug segmentation faults
+            "BENCHMARKS_DATA_DIR": f"{os.getenv('HOME')}/data",  # allows to avoid loading Python and R benchmarks input data from s3 for every build
+            "ARROWBENCH_DATA_DIR": f"{os.getenv('HOME')}/data",  # allows to avoid loading R benchmarks input data from s3 for every build
```

Comment:
This chunk runs all of the benchmarks together, so we need them together, yeah? Does this run the arrowbench ones a second time? 

Or I guess taking a step back: what's this code chunk helpful for | doing here?

Reply:
The env vars here are vestigial and unneeded (as of now). 

This dict is here because adapters will live in this repo (see `root` on line 80 is `arrow-benchmarks-ci/adapters`. We could stick them in a different repo if we want; it doesn't really matter. As of now the only one there is a mock for testing, but eventually at least the C++ and Go ones should live there.